lib/article_json/import/google_doc/html/image_parser.rb

```diff
@@ -36,17 +37,42 @@ def float
             super if floatable_size?
           end
 
+          # Extracts an href from the tag [image-link-to: url]) if present
+          # in the caption node.
+          # @return [String]
+          def href
+            return if @caption_node.nil?
+            match = @caption_node.content.strip.match(href_regexp)
+            return if match.nil?
+            remove_image_link_tag
+            match[:url]
+          end
+
           # @return [ArticleJSON::Elements::Image]
           def element
             ArticleJSON::Elements::Image.new(
               source_url: source_url,
               float: float,
-              caption: caption
+              caption: caption,
+              href: @href
             )
           end
 
           private
 
+          # Removes the [image-link-to: url] tag from the caption node
+          def remove_image_link_tag
+            @caption_node
+              .children
+              .first
+              .content = @caption_node.content.gsub(href_regexp, '').strip
+          end
+
+          # Regular expression to chech if there's a [image-link-to: url] tag
+          # @return [Regexp]
+          def href_regexp
+            %r{\[image-link-to: (?<url>.*)\]}
```

Comment:
I usually use `%r{\[image-link-to:\s+(?<url>.*)\]}` to make spaces a bit more flexible.

Reply:
cool, changing it!